docs/release-notes.7.0.md

```diff
@@ -1,5 +1,8 @@
 # Qooxdoo Release Notes
 
+## Fixes for v7.7.2
+For a full list of changes see https://github.com/qooxdoo/qooxdoo/commits/master?branch=master&qualified_name=refs%2Fheads%2Fmaster&since=2024-05-13&until=2024-19-07
```

Comment:
I would start where the last one left off

Reply:
What do you mean with last one?

Reply:
I mean 2024-05-02 since the last entry in the release notes notes 2024-05-02 as the until date for the release

https://github.com/qooxdoo/qooxdoo/commits/master?branch=master&qualified_name=refs%2Fheads%2Fmaster&since=2024-05-02&until=2024-19-07

Reply:
@oetiker I thought about including 2 May but some releases include previous version some don't. Is it done on purpose? If there is a rule to include prev release day I will include it. And about date for new release: It is a day when this PR will be merged and new release be published, but I don't know the date when this PR will be approved. How does it work? Could you explain please?

Reply:
Seems like I got how to do properly. I have to add changes in config files and this will be a date for new release. And then when PR will be approved and merged just publish via `qx pkg publish` to our package repository.